### DISCUSSION TITLE
Trabalhando em troca de equity

### DISCUSSION THREAD

**Discussion by rubenmarcus** [frontendbr/forum#590] [Discussão]
Pessoal,ando procurando bastante vagas lá fora, e vejo muitas (principalmente startups), ofercendo equity ( ações da empresa), e não salário.
Alguém já trabalhou nessa modalidade?
Sabe como funcionaria para estrangeiros e as garantias? Tem algumas startups e projetos que parecem bem interessantes,apesar do risco de trabalhar de graça.

**Comment by gabrieldarezzo** [comment]
Teve um podcast bem bacana do Hipsters, deram uma pincelada e comentaram um pouco as modalidades:

http://hipsters.tech/startups-venture-capital-e-investimentos-hipsters-45/

@vindi 
@peas

**Comment by guilhermehott** [comment]
Estou atualmente vivendo isso. Ainda nao sei se realmente vai dar certo
porque soh vou poder realizar meu lucro e vender minhas equities quando a
startup comecar a vende-las. Mas o que tentei fazer para me precaver foi um
modelo hibrido, recebo 1/5 do meu salario ideal, isso pagar minhas contas
(logico que vivendo gastando pouco), tenho alguns bonus programados e
aumentos de salarios para os proximos dois anos, de acordo com as rodadas
de investimento e alguns importantes milestones alcancados pelo produto.
Agora soh vou poder te dizer com certeza daqui uns 2 anos quando realmente
puder vender minhas equities.

On Wed, Jun 7, 2017 at 5:26 AM, Gabriel Darezzo <notifications@github.com>
wrote:

> Teve um podcast bem bacana do Hipsters, deram uma pincelada e comentaram
> um pouco as modalidades:
>
> http://hipsters.tech/startups-venture-capital-e-investimentos-hipsters-45/
>
> @vindi <https://github.com/vindi>
> @peas <https://github.com/peas>
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/frontendbr/forum/issues/590#issuecomment-306778918>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AEqSXEtiCzOqVmGlVh2kMfDUk2VZbMb1ks5sBpbbgaJpZM4NyUDP>
> .
>



-- 
*Guilherme Hott*
*Software Engineer*
Skype: guilhermehott
@guilhermehott
https://www.linkedin.com/in/guilhermehott

**Comment by munizart** [comment]
Invoco o @JackDaniells que trabalha num modelo parecido

**Comment by marcusvcribeiro** [comment]
Aproveitem e deem as dicas de onde conseguir trabalho assim.
Nem sabia que existia modelos de negócios assim.


2017-06-07 16:16 GMT-03:00 Artur Muniz <notifications@github.com>:

**Comment by rubenmarcus** [comment]
@marcusvcribeiro  fiz meu cadastro ontem no https://angel.co , recomendo um curriculo e portfolio em inglês , e boas skills de comunicação contam muitos pontos.

**Comment by rubenmarcus** [comment]
Mas comparando os salários anuais no angel.co em dolares, a maioria está na faixa dos 25k - 50k usd por ano pra um Front pleno. Até em paises do BRICS como Russia e India voce vagas nessa faixa. No Brasil as vagas estão abaixo de 15k usd

**Comment by felipeuntill** [comment]
Sou totalmente contra estes modelos de trabalho e vou lhes explicar os motivos.

Quando uma empresa lhe oferece uma oportunidade baseada em participação, você basicamente está trocando suas horas de trabalho quais possuem o valor hora que você receberia em mercado e está investindo este valor como esforço ou seja, se você ganha R$ 7.000,00 no mercado como arquiteto de software você está investindo todos os meses R$ 7.000,00 na empresa em questão.

Porém ai que entra o problema o salário médio de um profissional de marketing senior no Brasil fica em média R$  6.500,00(baseado nos contatos que tenho em São Paulo) um gerente comercial também fica em média R$ 7.000 + comissão.  

Baseado nos valores acima(quais não são métricas exatas mais servem para explicar meu ponto) é fácil constatar que ganhamos o mesmo ou mais do que estes profissionais ou seja nosso valor para a empresa é tão importante quanto o dos cargos comerciais + marketing e tudo isso é trocado por meros 3 ou 5%. Estes percentuais baixos não cativam o desenvolvedor por muito tempo fazendo com que eles desistam e deixem seu trabalho para trás.

Existe muita startup oferecendo participação já esperando a desistência mais contando que durante um breve período eles vão possuir um braço de TI para algumas tarefas.

Parem de sub valorizar o nosso trabalho. Uma startup é uma hipótese e uma hipótese nada mais é que porra nenhuma e como na maioria das vezes os fundadores são durangos e só investem tempo e esforço, se você está entrando com o mesmo você merece uma participação similar. 

Quero ver alguém contratar uma agência ou consultoria com 3%. Antes de aceitar uma proposta cote o valor do projeto em uma agência ou consultoria e reflita no quanto o vale o nosso esforço ai fora.

PS: Isso vale pro cenário **BR**, lá fora existe gente oferendo participação significativa em **`startup de verdade`** que **`já vale alguma coisa`**, não porque são bonzinhos mas porque lá fora profissional de TI é recurso escasso, caro e disputado a tapas.


_3% de #$ é #$%@ , sem mais..._

**Comment by rubenmarcus** [comment]
@felipeuntill  a empresa que estou me candidatando é a http://jump.careers/ . O CEO fez entrevista via skype e disse o mesmo que você. Me disse que hoje a empresa está avaliada em 3 milhões de dólares, e num futuro próximo pode passar por um round de investimentos de 10 milhões de dólares.

Pra mim parece furada também. O Cara te dá 3% da empresa e do dia pra noite você é dono de 300 mil dolares?

Mas pra mim é um desafio bacana, porque quero criar um portfolio meio que "internacional", e conseguir alguma vaga remota pra poder viajar, ou alguma vaga fora do Brasil, de preferencia na Ásia, por questões e gostos pessoais meus.

Mas coloquei na minha cabeça que em 2018 não estarei mais no Brasil e estarei ou trabalhando remoto, ou trabalhando para outra empresa local seja na Europa,Índia,China,Singapura,Tailândia,HongKong, não importa, quero ter esse conhecimento de trabalhar com equipes de fora do Brasil. Meu lance com a Ásia, além de pessoal, é pra sair um pouco do fluxo que todo mundo só pensa em EUA e Europa. Nada contra, talvez num segundo momento eu queira ir também, na verdade meu objetivo é ganhar minha independencia e qualidade de vida fora do Brasil.

**Comment by rubenmarcus** [comment]
Mas minha idéia é ganhar o mesmo que tiro no Brasil, mas em outras moedas, de preferência Euro ou Dolar, ou até menos, Vi vagas por ex: na Rep Tcheca que tenho muitos amigos brasileiros lá, devido a facilidade com o visto de PJ e de trabalho, e lá se paga o dobro do Brasil pra uma vaga com requisitos semelhantes dos daqui. E com o custo de vida lá valendo, meio que sobra um salário inteiro por mês, do que você ganha, dando pra planejar direitinho o que se pode gastar.

**Comment by felipeuntill** [comment]
Essas valuations baseadas em nada é o maior pega trouxa, se eu te falar que o computador que estou usando pra digitar agora vale um bilhão de dolares mas ninguem pagar isso, ele nunca valeu um bilhão simples assim.

@rubenmarcus Se for pra este proposito tá cheio de vagas nos seguintes sites:

http://www.vanhack.com/hackathon/
https://www.wfh.io
https://remote.com/
https://stackoverflow.com/jobs/
https://www.techifide.com/ (Empresa de Brasileiro em londres)

É melhor que trabalhar de graça.

**Comment by tarciozemel** [comment]
Em qualquer empreendimento corre-se "o risco de trabalhar de graça". Se você, depois de fazer suas análises, concluir que o negócio tem potencial, tem viabilidade (mesmo nunca tendo atuado nesta modalidade), acredito que seja interessante sim.

Muitas grandes empresas surgiram a partir de "somente uma hipótese".

**Comment by felipeuntill** [comment]
@tarciozemel O problema não está no conceito de startup e sim na quantidade equities oferecidas aos desenvolvedores.

Eu neste momento estou participando ativamente em 3 startups mas pra aceitar exigi mais que meros 3%.

**Comment by vortexchange** [comment]
> Pessoal,ando procurando bastante vagas lá fora, e vejo muitas (principalmente startups), ofercendo equity ( ações da empresa), e não salário.
> Alguém já trabalhou nessa modalidade?
> Sabe como funcionaria para estrangeiros e as garantias? Tem algumas startups e projetos que parecem bem interessantes,apesar do risco de trabalhar de graça.

Rubens, qual sua especialidade? Estamos em busca de um CTO para nossa plataform, caso se interesse entre em contato.

Vinicyus
+55 62 9 8140 0497
